backend/MeetHut.DataAccess/Entities/Meet/Room.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace MeetHut.DataAccess.Entities.Meet
+{
+    /// <summary>
+    /// Meet room entity
+    /// </summary>
+    public class Room : Entity
+    {
+        /// <summary>
+        /// Room name
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Public Id
+        /// </summary>
+        public string PublicId { get; set; }
+
```

Comment:
In my opinion, PublicId should be _required_.

Reply:
I do not remember the exact definition of the _PublicId_

Reply:
What do you remember @bencebaranyai @tothrobi

Reply:
It would stand for identifying the object in public interfaces instead of the sequential primary key.

Reply:
I trust in you. I will add required key.